cmd/helm/installer/install.go

```diff
@@ -30,21 +30,22 @@ import (
 	"k8s.io/client-go/kubernetes"
 	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	extensionsclient "k8s.io/client-go/kubernetes/typed/extensions/v1beta1"
+
 	"k8s.io/helm/pkg/chartutil"
 )
 
 // Install uses Kubernetes client to install Tiller.
 //
 // Returns an error if the command failed.
 func Install(client kubernetes.Interface, opts *Options) error {
-	if err := createDeployment(client.Extensions(), opts); err != nil {
+	if err := createDeployment(client.ExtensionsV1beta1(), opts); err != nil {
```

Comment:
It was announced at kubecon that the workloads API (deployments, replicasets, statefulsets, daemonsets, etc) will move to apps/v1 in 1.9, so this probably needs to be updated.

https://github.com/kubernetes/kubernetes/tree/v1.9.0-beta.2/staging/src/k8s.io/client-go/kubernetes/typed/apps/v1

Reply:
That would be a big breaking change for helm.  How long will extensions.Deployments be supported?

Reply:
I'm not sure how long it'll be supported. We should ask at the next SIG-Apps meeting.

Just to clarify, when you mean by breaking change, you mean that `helm init --upgrade` will break by moving to apps/v1, right?

Reply:
From #sig-apps:

> [minimum] 2 releases or 6 months after announced deprecation. See https://kubernetes.io/docs/reference/deprecation-policy/

Reply:
IMO I think we should move to `apps/v1`. Anyone who has upgraded their Kubernetes cluster to 1.9 should know about that switch and we can make it clear in the release notes that for this upgrade they'll need to delete the old Tiller

Reply:
Yeah, we need to wait, especially as the adoption cycle for new Kubernetes versions is slowing down. (We still see a lot of 1.6 and even 1.5 in prod)

Reply:
I agree.  I will leave it as it.